dockers/docker-ptf/Dockerfile.j2

```diff
@@ -96,8 +96,11 @@ RUN mkdir /var/run/sshd \
  && sed -ri 's/UsePAM yes/#UsePAM yes/g' /etc/ssh/sshd_config \
  && sed -i '$aUseDNS no' /etc/ssh/sshd_config
 
-COPY ["supervisord.conf", "sshd.conf", "ptf_nn_agent.conf", "/etc/supervisor/conf.d/"]
+COPY ["supervisord.conf", "/etc/supervisor/"]
+COPY ["conf.d/supervisord.conf", "conf.d/sshd.conf", "conf.d/ptf_nn_agent.conf", "/etc/supervisor/conf.d/"]
+
+RUN mkdir -p /var/log/supervisor
 
 EXPOSE 22
 
-ENTRYPOINT ["/usr/bin/supervisord"]
+ENTRYPOINT ["/usr/local/bin/supervisord", "-c", "/etc/supervisor/supervisord.conf"]
```

Comment:
There is no /etc/supervisor.conf file.
There're:
/etc/supervisor/supervisord.conf file with full configuration
/etc/supervisor/conf.d/supervisord.conf with one-line adjustment.
I use "-c /etc/supervisor/supervisord.conf" which loads three other configurations with include.

Output from log
2017-07-24 20:53:00,652 CRIT Supervisor running as root (no user in config file)
2017-07-24 20:53:00,652 INFO Included extra file "/etc/supervisor/conf.d/ptf_nn_agent.conf" during parsing
2017-07-24 20:53:00,652 INFO Included extra file "/etc/supervisor/conf.d/sshd.conf" during parsing
2017-07-24 20:53:00,652 INFO Included extra file "/etc/supervisor/conf.d/supervisord.conf" during parsing
2017-07-24 20:53:00,665 INFO RPC interface 'supervisor' initialized
2017-07-24 20:53:00,665 CRIT Server 'unix_http_server' running without any HTTP authentication checking
2017-07-24 20:53:00,665 INFO supervisord started with pid 7
2017-07-24 20:53:01,667 INFO spawned: 'sshd' with pid 10
2017-07-24 20:53:01,669 INFO spawned: 'ptf_nn_agent' with pid 11
2017-07-24 20:53:02,670 INFO success: sshd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2017-07-24 20:53:02,671 INFO success: ptf_nn_agent entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
root@24850f76d0a1:/# supervisorctl status
ptf_nn_agent                     RUNNING   pid 11, uptime 0:01:00
sshd                             RUNNING   pid 10, uptime 0:01:00